src/components/tabsv2/TabsV2.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { FunctionComponent } from "react";
+import styled, { css } from "styled-components";
+import theme from "../../theme";
+import { Text } from "../text";
+import { Box } from "../box";
+import { Button } from "../button";
+
+type TabProps = {
+  className?: string;
+  wizard?: boolean;
+  step?: number;
+  heading?: React.ReactNode;
+  caption?: string;
+  activeTab?: number;
+  children?: React.ReactNode;
+  onClick?: (e: any) => void;
+  onChange?: (e: any) => void;
+  props?: any;
+  styleActive?: boolean;
+  index?: number;
+  disabled?: boolean;
+  buttons?: React.ReactNode;
+  buttonsJustify?: string;
+};
+
+type TabsProps = {
+  wizard?: boolean;
+  name?: string;
+  caption?: string;
+  children: Array<TabProps>;
+  handleTab: (e: any) => void;
+  step: number;
+};
+
+type ButtonPreviousProps = {
+  onClick?: (e: any) => void;
+};
+
+type ButtonNextProps = {
+  onClick?: (e: any) => void;
+};
+
+export const TabsV2Content = styled(Box)`
+  background-color: ${theme.color.background.ui01};
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  box-shadow: none;
+  padding: ${theme.spacing.spacing06} ${theme.spacing.spacing04};
+  p {
+    line-height: 1.5rem;
+    &:first-child {
+      margin-top: 0;
+    }
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+`;
+
+export const TabV2Container: any = styled.div<TabProps>`
+  padding: ${theme.spacing.spacing04};
+  background-color: ${theme.color.background.ui03};
+  border-right: 1px solid ${theme.color.border.border01};
+  border-bottom: 1px solid transparent;
+  width: 17rem;
+  &:last-child {
+    border-right: none;
+    &.tab-active {
+      border-right: 1px solid ${theme.color.border.border01};
+    }
+  }
+
+  &:hover {
+    cursor: ${({ disabled }) => (disabled ? `auto` : `pointer`)};
+    background-color: ${({ disabled }) =>
+      disabled
+        ? `${theme.color.background.ui03}`
+        : `${theme.color.background.ui01}`};
+  }
+
```

Comment:
All thesse types should be exported from `Tabs.tsx` as well

Reply:
Yes, Modestas, I have resued most type and style from Tabs and some style  `TabV2Container` need update using new component, so I recode it, any question please let me know, Thx